features/delete_object.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Delete a content object
+
+  Background:
+    Given I am logged in
+    And I have the "pre_release_features" permission
+    And the organisation "Ministry of Example" exists
+    And a schema "pension" exists:
+    """
+    {
+       "type":"object",
+       "required":[
+          "description"
+       ],
+       "additionalProperties":false,
+       "properties":{
+          "description": {
+            "type": "string"
+          }
+       }
+    }
+    """
+    And the schema has a subschema "rates":
+    """
+    {
+      "type":"object",
+      "required": ["title", "amount"],
+      "properties": {
+        "title": {
+          "type": "string"
+        },
+        "amount": {
+          "type": "string",
+          "pattern": "£[0-9]+\\.[0-9]+"
+        },
+        "frequency": {
+          "type": "string",
+          "enum": [
+            "a week",
+            "a month"
+          ]
+          },
+          "description": {
+            "type": "string"
+          }
+      }
+    }
+    """
+
+  Scenario: GDS editor creates a Pension and deletes it
+    When I visit the Content Block Manager home page
+    And I click to create an object
+    And I click on the "pension" schema
+    And I complete the form with the following fields:
+      | title            | description   | organisation        | instructions_to_publishers |
+      | my basic pension | this is basic | Ministry of Example | this is important  |
+    And I save and continue
+    And I confirm my answers are correct
+    And I save a draft
```

Comment:
Perhaps it's not necessary to drive through these preliminaries? It might be sufficient to go for a starting state where we're looking at a draft edition?

I think that be done with:

```gherkin
Scenario: GDS editor creates a Pension and deletes it
  Given a pension content block has been drafted
  And I am viewing the draft edition
```

Reply:
Ooh that's nice. I did not know that existed. Certainly cuts down on the cruft